utils/land-patterns.stanza

```diff
@@ -301,6 +435,7 @@ public pcb-landpattern dummy-landpattern (n:Int, courtyard:[Double,Double]):
 ; X - pad height
 ; w, h - width and height of courtyard in mm
 public defn make-two-pin-landpattern (Z:Double, G:Double, X:Double, w:Double, h:Double, courtyard?:True|False, polarized?:True|False) :
+  println("making two pin landpattern")
```

Comment:
Don't leave print statements in production code.

Reply:
removed